examples/inputs.rs

```diff
@@ -0,0 +1,19 @@
+/// A small demonstration of the Input API.
+/// This prints embedded bytes with a custom header and then reads from STDIN.
+use bat::{Input, PrettyPrinter};
+
+fn main() {
+    PrettyPrinter::new()
+        .header(true)
+        .grid(true)
+        .line_numbers(true)
+        .inputs(vec![
+            Input::from_bytes(b"echo 'Hello World!'")
+                .name("embedded.sh")
+                .title("An embedded shell script.")
+                .kind("Embedded"),
```

Comment:
minor: maybe we could call `.kind` before `.title` in the same way as the output order?

The `.name(…)` part here is needed for syntax detection? Maybe we could leave it out and add `#!/bin/bash` instead? Or add a comment why the dummy name is needed?

In the future, we could probably also replace it with `.language(…)`.

Reply:
Yeah, the name is needed for syntax detection. In the future we definitely should add a `.language` option to replace it, though.